### PR TITLE
Tabs should preview on tab page hover

### PR DESCRIPTION
**Pull request by bbondy**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

### CODE REVIEW

**Comment by bbondy** [comment]
Auditors: @diracdeltas

**Comment by bbondy** [comment]
Going to merge this bug pls review and I'll do follow ups for it.

File: app/sessionStore.js [hunk 1]
```diff
@@ -97,6 +97,10 @@ module.exports.cleanPerWindowData = (perWindowData) => {
   delete perWindowData.contextMenuDetail
   // Don't save preview frame since they are only related to hovering on a tab
   delete perWindowData.previewFrameKey
+  // Don't save preview tab pages
+  if (perWindowData.ui && perWindowData.ui.tabs) {
+    delete perWindowData.ui.tabs.previewTabPageIndex
```
Comment (diracdeltas) [inline app/sessionStore.js]: should be added to docs/state.md

**Comment by diracdeltas** [comment]
++

**Pull request merged**